env/wflow_odin

```diff
@@ -0,0 +1,47 @@
+#%Module
+
+proc ModulesHelp { } {
+   put stderr "This module loads python environement for running SRW on"
+   put stderr "the NOAA RDHPC machine Hera"
+}
+
+module-whatis "Loads libraries needed for running SRW on Hera"
+
+if { [module-info mode load] } {
+  # >>> conda initialize >>>
+  # !! Contents within this block are managed by 'conda init' !!
+  set cmd_init {\
+__conda_setup="$('/scratch/software/Odin/python/anaconda2/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
+if [ $? -eq 0 ]; then
+    eval "$__conda_setup"
+else
+    if [ -f "/scratch/software/Odin/python/anaconda2/etc/profile.d/conda.sh" ]; then
+        . "/scratch/software/Odin/python/anaconda2/etc/profile.d/conda.sh"
+    else
+        export PATH="/scratch/software/Odin/python/anaconda2/bin:$PATH"
+    fi
+fi
```

Comment:
I am not sure this section would work. It looks fairly bash-like.

It may be useful for someone to install a module to handle this Python installation. There seems to be no way to "unload" it and that can lead to issues on some platforms.

Reply:
Yeah, wflow_odin seem to be the most challenging. TCL language allows to call shell scripts or commands via a puts to stdout so was just hoping it will work. Recent modules versions have `sh-to-mod`, but couldn't find it on any system to test ...